content/troubleshooting/common-ios-issues.md

```diff
@@ -226,3 +226,22 @@ Apple has updated Apple Developer Program License Agreement and it needs to be r
 
 ###### Solution
 In order to update your existing apps and submit new apps to the App Store, the Account Holder must review and accept the updated agreement by signing in to their [account](https://appstoreconnect.apple.com/agreements/#/) on the Apple Developer website.
+
+### Certificate will no longer be valid in 30 days
+
+###### Description
+You received e-mail from Apple stating that:
+
+    Your Distribution Certificate will no longer be valid in 30 days. To generate a new certificate, sign in and visit Certificates, Identifiers & Profiles.
+
+###### Cause
+Distribution certificates issued by Apple have a validity period of one year from the date of issuance. After one year, they expire, which means they can no longer be used to sign and distribute apps.
+
+###### Solution
+Rest assured, Codemagic simplifies the certificate management process for you. Here's how it works:
+ - If you're using `automatic code signing`, Codemagic takes care of certificate and provisioning profile management for you. This means it also automatically renews the certificate when it's about to expire.
+
+ - If you're using `manual code signing` or `Codemagic signing identities (CSI)`, you will need to create and upload your certificates and profiles that you generated on the Apple Developer UI manually to Codemagic.
+ Alernativley, with CSI, users can also fetch or create resources directly from Codemagic UI instead of navigating the Apple Developer UI when going in **Teams** > **Codesignng identities** > **iOS certificate** tab > **Generate certificate** button and in there provide your API key.
+
+     If you want to use a specific certificate with CSI, you'll need to provide its private RSA key associated with an iOS Distribution certificate that has previously been generated within your Apple Developer Program account. You can find additional information on how to update an RSA key (CERTIFICATE_PRIVATE_KEY) for a certificate [here](https://docs.codemagic.io/yaml-code-signing/alternative-code-signing-methods/#:~:text=Obtaining%20the%20Certificate%20private%20key).
```

Comment:
providing RSA key is not needed when using CSI, it is necessary when using automatic signing.